src/open_inwoner/pdc/tests/test_product_location.py

```diff
@@ -227,3 +227,48 @@ def test_exception_is_handled_when_city_and_postcode_are_not_provided(
             ],
         )
         mock_geocoding.assert_called_once_with(" ,  ")
+
+
+class TestLocationDetailView(WebTest):
+    def test_published_product_is_rendered(self):
+        product = ProductFactory()
+        product_location = ProductLocationFactory()
+        product.locations.add(product_location)
+
+        response = self.app.get(
+            reverse("pdc:location_detail", kwargs={"uuid": product_location.uuid})
+        )
+
+        self.assertEqual(response.context["products"].get(), product)
+
+    def test_locations_details_are_rendered(self):
+        product = ProductFactory()
+        product_location = ProductLocationFactory(
```

Comment:
We should add email and phonenumber to the factory?

(for a Dutch phonenumber I used `phonenumber = factory.Faker("numerify", text="0#########")`, because the regular phonenumber provider has extra characters)

Reply:
Here I follow the wiki's best practices which says not to define non-required fields. But if you think I should, I will of course add it.

Reply:
The wiki is more of a starting point or guideline you can ignore if you can make a defendable case for the transgression. 

I think factories should be useful and help get usable test data.